libs/types/src/tokens.rs

```diff
@@ -210,6 +199,61 @@ pub struct CustomMetadata {
 
 	/// Whether an asset can be used as a currency to fund Centrifuge Pools.
 	pub pool_currency: bool,
+
+	/// The ways, if any, this token is cross-chain transferable
+	pub transferability: Option<CrossChainTransferability>,
+}
+
+impl Default for CustomMetadata {
+	fn default() -> Self {
+		Self {
+			xcm: Default::default(),
+			mintable: false,
+			permissioned: false,
+			pool_currency: false,
+			transferability: Some(CrossChainTransferability::Xcm),
+		}
+	}
+}
```

Comment:
I mean, by default tokens are transferable. The only exception to that, to my knowledge, is the `CurrencyId::Staking` tokens. And within that, XCM is the default because it's the general status quo in Polkadot, whereas Connectors is an ad-hoc, Centrifuge-specific solution.

In practise, this is most useful for testing indeed, because this default is never imposed anywhere else to my knowledge.

Reply:
See above :)